sonic_api.yaml

```diff
@@ -974,7 +974,7 @@ paths:
 #----------------------------------------------
   '/config/vrouter/{vnet_id}/routes':
     patch:
-      summary: Update/modify IP routes for a virtual network router
+      summary: Update/modify IP routes for a virtual network router. Note: Modifying route is not currently supported, instead delete and add
```

Comment:
I think we should avoid the usage of `update` or `modify` to prevent confusion. Instead, we could say `add or delete IP routes for a virtual network router`  which is all that we currently support.